presto-base-arrow-flight/src/main/java/com/facebook/plugin/arrow/ArrowMetadata.java

```diff
@@ -192,6 +195,12 @@ public Map<SchemaTableName, List<ColumnMetadata>> listTableColumns(ConnectorSess
         return columns.build();
     }
 
+    @Override
+    public String normalizeIdentifier(ConnectorSession session, String identifier)
+    {
+        return arrowFlightConfig.isCaseSensitiveNameMatching() ? identifier : identifier.toLowerCase(ENGLISH);
```

Comment:
If it connector can support unicode and not english then I think we should use `Locale.ROOT`